packages/sdk-v2/src/contexts/Web3Context.tsx

```diff
@@ -105,18 +105,18 @@ export const Web3Provider = ({ children, config, web3Provider, switchNetworkRequ
   const [switchNetwork, setSwitchNetwork] = useState<SwitchCallback>();
 
   const setSwitcNetworkCallback = (cb: SwitchCallback) => setSwitchNetwork(() => cb);
-
   //make sure we have Fuse and mainnet by default and the relevant multicall available from useConfig for useMulticallAtChain hook
   config.networks = [Fuse, Mainnet, Kovan, Ropsten, Goerli, Celo, ...(config.networks || [])];
   config.multicallVersion = config.multicallVersion ? config.multicallVersion : 1;
+  config.gasLimitBufferPercentage = 10;
```

Comment:
Relying on default gas limit occasionally reverted because 'out of gas'
so this adds a buffer of 10 percent, and I have not had a single reverted transaction